travis/test_pylint

```diff
@@ -9,12 +9,54 @@ import os
 
 import run_pylint
 
+from getaddons import get_modules_changed
 
 pylint_rcfile = os.path.join(
     os.path.dirname(os.path.realpath(__file__)),
     'cfg',
     "travis_run_pylint.cfg")
-result = run_pylint.main([
+count_errors = run_pylint.main([
     "--config-file=" + pylint_rcfile,
 ], standalone_mode=False)
-exit(result)
+
+
+pylint_rcfile_pr = os.path.join(
+    os.path.dirname(os.path.realpath(__file__)),
+    'cfg',
+    "travis_run_pylint_pr.cfg")
+
+is_pull_request = os.environ.get(
+    'TRAVIS_PULL_REQUEST', 'false') != 'false'
```

Comment:
We are using a variable of travis app... We don't have control in this one.
But I copied the use of this variable of other stable code in MQT:
[is_pull_request](https://github.com/OCA/maintainer-quality-tools/blob/87f2b269667c02af210493e929eb9524eb455907/travis/travis_run_tests#L57-L58)